src/analytics/formResponseTracker/formResponseTracker.test.ts

```diff
@@ -26,6 +31,8 @@ describe("form with multiple fields", () => {
       '  <select id="region" name="region"><option value="test value">test value</option><option value="test value2" selected>test value2</option></select>' +
       '  <label for="username">text input section</label>' +
       '  <input type="text" id="username" name="username" value="test value"/>' +
+      '  <label for="password">password input section</label>' +
+      '  <input type="password" id="password" name="password" value="test gregre value"/>' +
```

Comment:
Is "gregre" filler text?

Reply:
very very minor but just thinking so it’s consistent we input 'lorem ipsum'

Reply:
You don't like my password?!?! :-0 :-)
Value is not important, can be anything. Because at the end, data layer event text param needs to be "undefined"

Reply:
Okk i seeee